docs/quality-assurance-test-cases.md

```diff
@@ -88,6 +88,18 @@ This document contains some examples of test cases for each feature implemented
 | Email in request body does not belong to a registered User                  | Fail    |
 | Email in request body is from a Verified User                               | Fail    |
 
+### Refresh token
+
+**Service:** POST /user/refresh
+
+| Test Case                                                        					| Outcome |
+|-----------------------------------------------------------------------------------|---------|
+|Token in request body is the refresh token got after login request                 | Success |
```

Comment:
```suggestion
| Refresh token in Authorization field is the refresh token returned on login response                 | Success |
```

Reply:
This is because if you notice in the Swagger UI there is no request body:

![image](https://user-images.githubusercontent.com/11148726/76699692-ea9dc080-66a7-11ea-9015-ca9935fd2b01.png)